### DISCUSSION TITLE
Good Code Workflows

### DISCUSSION THREAD

**Discussion by ghost** [Bijoujs/Bijou.js#69] [Q&A] [answered]
These code cleaning workflows are great! How do I set them up?

**Comment by ghost** [comment]
um anyone there

**Reply by ghost** [reply]
just saying @Explosion-Scratch

**Comment by Explosion-Scratch** [comment, marked as answer]
Oh, they're sort of complicated, what I did is I searched [GitHub marketplace](https://github.com/marketplace?type=actions) actions for the stuff I needed, and a bunch of stackoverflow stuff. You can learn more [here](https://docs.github.com/en/actions/creating-actions)
